iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -43,17 +43,29 @@
 
 #define DEBUG_TYPE "iree-flow-dispatch-linalg-on-tensors"
 
+// TODO(ravishankarm): Prune this list. These flags should go away ASAP!!
+
 static llvm::cl::list<int64_t> clLinalgOnTensorsTileSizes(
     "iree-flow-dispatch-linalg-on-tensors-tile-sizes",
     llvm::cl::desc("Comma-separated list of tile sizes for tiling on tensors"),
     llvm::cl::CommaSeparated);
 
+// TODO(#5040): This works for the most part but the downstream bufferization
```

Comment:
Ah this is independent from enabling tile-and-distribute for element wise ops but was triggered by disabling it ?

Reply:
Yeah, just adding comments here that these options should be removed soon.